site/source/docs/getting_started/downloads.rst

```diff
@@ -47,6 +49,43 @@ Run the following :ref:`emsdk <emsdk>` commands to get the latest tools from Git
 
 If you change the location of the SDK (e.g. take it to another computer on an USB), re-run the ``./emsdk activate latest`` and ``source ./emsdk_env.sh`` commands.
 
+Emsdk install targets
+---------------------
+
+In the description above we asked the emsdk to install and activate ``latest``, which is the latest tagged release. That is often what you want.
+
+You can also install a specific version by specifying it, for example,
```

Comment:
I think you can merge the `::` with the end of the last line of text (here and below)

Reply:
Not sure what you mean by "merge" here?

(This seems to be the pattern we use elsewhere, a new line, then `  ::` then a new line, then the code?)

Reply:
In restructuredtext, you can (should?) end the preceeding paragaph with a `::`.   I think it looks a little neater and is more compact, but its really nit pick

See  http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#literal-blocks

Reply:
Interesting, I didn't know that.

Looks like doing it that way changes the indentation, it unindents it compared to what we have now. Not sure which is better, but we should probably be consistent everywhere?

Reply:
In any case that seems separate - I can open an issue for that if you think it's worth looking into?

Reply:
No worries.  I agree we should be consistent.